trestle/transforms/transformer_registry.py

```diff
@@ -0,0 +1,47 @@
+# -*- mode:python; coding:utf-8 -*-
+# Copyright (c) 2020 IBM Corp. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Define and register all transformers here."""
```

Comment:
I think we need to register tranformers here not define them?

Reply:
I thought all transformers would be defined and registered in one place - but we can instead just create the singleton here and define/register them in totally separate files.  As it is these are just stand-ins for the real things - so they can simply be removed when we have transformers in place.